Tests/fixtures/podcasting.yml

```diff
@@ -0,0 +1,9536 @@
+    platform: ""
+  client:
+    type: mobile app
+    name: Yapa
+    version: "1.11.32"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'yapa/561 CFNetwork/1406.0.4 Darwin/22.4.0'
+  os:
+    name: iOS
+    version: "16.4"
+    platform: ""
+  client:
+    type: mobile app
+    name: Yapa
+    version: "561"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Zeno'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Zeno
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Zune/4.8'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Zune
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Libsyn4-peek'
+  os: [ ]
+  client: null
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'UCast/1 CFNetwork/1406.0.4 Darwin/22.4.0'
+  os:
+    name: iOS
+    version: "16.4"
+    platform: ""
+  client:
+    type: mobile app
+    name: UCast
+    version: "1"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'NPROneAndroid'
+  os: [ ]
+  client:
+    type: mobile app
+    name: NPR One
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'NPR%20One/371 CFNetwork/1399 Darwin/22.1.0'
+  os:
+    name: iOS
+    version: "16.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: NPR One
+    version: "371"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Trackable/0.1 NNjCeA'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Trackable
+    version: "0.1"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Uforia/1056 CFNetwork/1406.0.4 Darwin/22.4.0'
+  os:
+    name: iOS
+    version: "16.4"
+    platform: ""
+  client:
+    type: mobile app
+    name: Uforia
+    version: "1056"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'LAT-Native-App; iPhone; iOS 15.7.3'
+  os:
+    name: iOS
+    version: "15.7.3"
+    platform: ""
+  client:
+    type: mobile app
+    name: L.A. Times
+    version: ""
+  device:
+    type: smartphone
+    brand: Apple
+    model: 'iPhone'
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Mozilla/5.0 (iPhone; CPU iPhone OS 16_1_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) nytios/9.91.1'
+  os:
+    name: iOS
+    version: "16.1.2"
+    platform: ""
+  client:
+    type: mobile app
+    name: NYTimes
```

Comment:
I believe this refers https://apps.apple.com/us/app/the-new-york-times/id284862083 so the `The New York Times` should be used as name

Reply:
They have given me some specific instructions to name it `NYTimes`. I just added an older distinct podcast app of their `NYT Audio`. They want them different because they want to track the conversion of users of the old app vs the new app. I know that their app is called 'The New York Times', and I know that that is everywhere else on all their content, only their domain name is 'www.nytimes.com'.

Because their podcast is the most popular podcast in the world, with billions of downloads, I have to take into account their recommendation, even if I disagree with it.

But for what it is worth, since their applications only serve their own content, this only affects us and no one else who uses this library will likely ever see this user agent, so can I have this one? If not, I will change it here, and merely have that singular divergence in our rust library.

Reply:
Personally, I do not mind. If this was a request from someone from NYTimes themselves, it may bee a good idea to have it as evidence somewhere here for traceability (like a screenshot with redacted PII if any), but that's optional

Reply:
After looking at the tickets, I think I'm going to just call this here, `The New York Times`. It is definitely the same app, they just had a different user agent for iOS.

They are getting into the weeds to split their new and old IOS app user agents, and I don't think the distinction belongs in this library. I will keep that difference in the rust library, because it is so low impact, and because that is the behavior they expect from us.